src/handlers/wildcard/unassign.ts

```diff
@@ -41,8 +39,10 @@ const checkBountyToUnassign = async (issue: any): Promise<boolean> => {
   const curTimestamp = new Date().getTime();
 
   if (curTimestamp > deadLineOfIssue) {
-    log.debug(`Releasing the bounty back to dev pool because the allocated duration already ended`, { deadLineOfIssue, curTimestamp });
-    await addCommentToIssue(`Releasing the bounty back to dev pool because the allocated duration already ended`);
+    log.debug(
+      `Releasing the bounty back to dev pool because the allocated duration already ended! deadLineOfIssue: ${deadLineOfIssue}, curTimestamp: ${curTimestamp}`
+    );
+    await addCommentToIssue(`Releasing the bounty back to dev pool because the allocated duration already ended`, issue.number);
```

Comment:
It could be nice to have a global strings config for projects to be able to update these messages. It's not a priority and we would certainly be fine to operate without this feature but I'll make an issue for it anyways.

Reply:
Totally agree!